litmus-portal/frontend/src/views/Analytics/WorkflowDashboard/WorkflowComparisonTable/index.tsx

```diff
@@ -286,7 +286,7 @@ const WorkflowComparisonTable = () => {
               console.error(error);
             }
           });
-          if (isWorkflowValid) {
+          if (isWorkflowValid && selectedRuns.length) {
```

Comment:
For a recurring workflow, may we show an empty graph or inform the user that there is not data

Reply:
Yes we are doing that, there is already a message as no runs available. @whitetiger1399